ramlfications/loader.py

```diff
@@ -35,7 +42,88 @@ def _yaml_include(self, loader, node):
                 return inputfile.read()
 
         with open(file_name) as inputfile:
-            return yaml.load(inputfile, self._ordered_loader)
+            parsed = yaml.load(inputfile, self._ordered_loader)
+
+            if file_ext == ".json":
+                parsed = self._parse_json_refs(
+                    parsed,
+                    os.path.dirname(file_name),
+                )
+
+            return parsed
+
+    def _lookup_json_ref(self, ref_key, base_path=None, parent_schema=None):
+        """
+        Traverses the json pointer and returns the value of the pointer.
+        """
+        if "#" not in ref_key:
```

Comment:
@benhamill so this doesn't differentiate between internal & external, because both are valid:

```json
{
    "$ref": "http://foo.example.com/exampledata#properties"
}
```
will pull from:

```json
{
  "properties": {
    "name": ...
  }
}
```
and this would work as well too:
```json
{
  "$ref": "#items",
  "items": ["some", "items"]
}
```

However I did get the [jsonref](https://github.com/gazpachoking/jsonref) package to work for local relative files, local absolute files, remote files, all with and without fragments, so my soon-to-be-uploaded patch has incorporated it.